conformance/src/test/java/build/buf/protovalidate/ValidatorTest.java

```diff
@@ -139,7 +141,8 @@ public void strictWrapperDouble() throws Exception {
 
   @Test
   public void strictFieldExpressions() throws Exception {
-    FieldExpressions invalid = FieldExpressions.newBuilder().build();
+    FieldExpressionMultipleScalar invalid =
```

Comment:
The message was replaced by several individual messages in https://github.com/bufbuild/protovalidate/pull/329.